File: typescript/dataBasicTest.ts

function test() {
	var a = 10
	var a0 = 10.99
	var a1 = true
	var a2 = "apple"
	console.log(a, a0, a1, a2)
}

function test2() {
	var a: number = 10
	var a0: number = 10.99
	var a1: boolean = true
	var a2: string = "apple"
	var a3: any = null
	var a4: any
	console.log(a, a0, a1, a2, a3, a4)
}

test2()

File: python/dataBasicTest.py

def test():
	a0 = 10
	a1 = 10.25
	a2 = "apple"
	a3 = True
	a4 = 10+3j

	print(a0, type(a0), isinstance(a0, int))
	print(a1, type(a1), isinstance(a1, float))
	print(a2, type(a2), isinstance(a2, str))
	print(a3, type(a3), isinstance(a3, bool))
	print(a4, type(a4), isinstance(a4, complex))

test()